cltk/corpus/arabic/alphabet.py

```diff
@@ -0,0 +1,16 @@
+"""Arabic alphabet"""
+
+
+#Arabic digits from 0 to 9
```

Comment:
The Hindu-Arabic numerals (the ones you linked to) were invented in India. They were later learned by the Arab world; the Arab world taught them to the rest of the world. Hence, an Arabic was appended (Hindu-Arabic). In the classical arabic script -- feel free to prove me wrong -- the Hindu-Arabic digits were not used. What was used was a different set of digits for the same numbers, which you correctly called Eastern Arabic Numerals. Since there was just one list, I did not care to make the distinction.
@kylepjohnson what do you think should be done in this case?